modules/test/playwright/pages/roles-admin-web/RoleDefinePermissionsPage.ts

```diff
@@ -27,12 +31,45 @@ export class RoleDefinePermissionsPage {
 			return page.getByTestId(id).getByRole('menuitem');
 		};
 		this.page = page;
+		this.permissionCheckbox = (permissionName: string) => {
+			return page.getByLabel(permissionName);
+		};
 		this.portletResourceLabel = page.getByTestId('portletResourceLabel');
 		this.resourceSection = (title: string) => {
 			return page
 				.locator('.sheet-tertiary-title')
 				.filter({hasText: title});
 		};
+		this.saveButton = page.getByRole('button', {name: 'Save'});
```

Comment:
`this.saveButton = page.getByRole('button', {**exact: true,** name: 'Save'});`

Reply:
The `exact` prop will help here if on the page there is another button with a label that starts with "Save"